src/app/view/components/toolbar-add-contentlet/toolbar-add-contentlet.html

```diff
@@ -1,15 +1,17 @@
 <dot-dropdown-component
+    #dropdown
```

Comment:
I think we don't need this, it was just for the test, right?

Reply:
Right, I'm removing that.